test/csuite/wt7989_compact_checkpoint/main.c

```diff
@@ -166,12 +169,8 @@ run_test(bool stress_test, const char *home, const char *uri)
     printf(" - Compressed file size MB: %f\n - Original file size MB: %f\n",
       file_sz_after / (1024.0 * 1024), file_sz_before / (1024.0 * 1024));
 
-    /*
-     * FIXME-WT-8055 At the moment the assert below is commented out to prevent evergreen from going
-     * red. Please enable the assert as soon as the underlying defect is fixed and compact does its
-     * job well.
-     */
-    /*testutil_assert(file_sz_before * 0.9 > file_sz_after);*/
+    /* Make sure the compact operation has reduced the file size by at least 20%. */
+    testutil_assert(file_sz_before * 0.8 > file_sz_after);
```

Comment:
Using floating point operations on uint64_t seems a little weird to me, but it should work. OK, just getting over it... :)

Reply:
Changed to integer calculations: `(file_sz_before / 100) * 80 > file_sz_after`. Rounding error of up to 7920 bytes is acceptable giving the file size is 1378271232 bytes.